### PR TITLE
Fix stat functions with no numeric columns.

### PR DESCRIPTION
**Pull request by ueshin**
Some statistic functions fail if there are no numeric columns.

```py
>>> kdf = ks.DataFrame({"A": pd.date_range("2020-01-01", periods=3), "B": pd.date_range("2021-01-01", periods=3)})
>>> kdf.mean()
Traceback (most recent call last):
...
ValueError: Current DataFrame has more then the given limit 1 rows. Please set 'compute.max_rows' by using 'databricks.koalas.config.set_option' to retrieve to retrieve more than 1 rows. Note that, before changing the 'compute.max_rows', this operation is considerably expensive.
```

The functions which allow non-numeric columns by default are:

- `count`
- `min`
- `max`

### CODE REVIEW

File: databricks/koalas/tests/test_stats.py [hunk 1]
```diff
@@ -27,35 +27,46 @@
 
 
 class StatsTest(ReusedSQLTestCase, SQLTestUtils):
-    def _test_stat_functions(self, pdf, kdf):
+    def _test_stat_functions(self, pdf_or_pser, kdf_or_kser):
```
Comment (xinrong-databricks) [inline databricks/koalas/tests/test_stats.py:30]: Great refactoring!

**Review by xinrong-databricks — APPROVED** [review]

**Comment by xinrong-databricks** [comment]
LGTM! Thank you!

**Comment by xinrong-databricks** [comment]
Shall we specify which statistic functions support non-numeric columns in the PR description?

**Review by itholic — APPROVED** [review]
LGTM

**Review by HyukjinKwon — APPROVED** [review]

**Pull request merged**